.github/workflows/download_stats.yml

```
@@ -42,30 +42,24 @@ jobs:
           git add assets/downloads.svg
 
           if ! git diff --cached --quiet; then
-            git checkout -b chore/update-download-stats
+            BRANCH="chore/update-download-stats-$(date -u +%Y%m%d%H%M%S)"
+            echo "branch=$BRANCH" >> "$GITHUB_OUTPUT"
+            git checkout -b "$BRANCH"
             git commit -m "update download stats"
           fi
 
       - name: push branch and open PR
+        if: steps.commit.outputs.branch != ''
```

actionlint (1.7.12)

[error] property "commit" is not defined in object type {}

(expression)